File: official/docs/csharp/scanform/CreateScanform.cs
using EasyPost;

EasyPost.ClientManager.SetCurrent("EASYPOST_API_KEY");

ScanForm scanForm = await ScanForm.Create(new List<Dictionary<string, object>>(){
    {"id", "shp_..."},
    {"id", "shp_..."}
});

Console.Write(scanForm);

File: official/docs/php/scanforms/CreateScanform.php
<?php

\EasyPost\EasyPost::setApiKey($_ENV['EASYPOST_API_KEY']);

$scanForm = \EasyPost\ScanForm::create([
    'shipments' => [
        ['id' => 'shp_...'],
        ['id' => 'shp_...'],
    ]
]);

echo $scanForm;
